File: 3512-minimum-operations-to-make-array-sum-divisible-by-k/3512-minimum-operations-to-make-array-sum-divisible-by-k.cpp
class Solution {
public:
    int ArraySum(vector<int>& arr){
        int s=0;
        for(int i=0;i<arr.size();i++){
            s+=arr[i];
        }
        return s;
    }
    int minOperations(vector<int>& nums, int k) {
        int SumofElements = ArraySum(nums);
        return(SumofElements % k);
    }
};

File: 3512-minimum-operations-to-make-array-sum-divisible-by-k/3512-minimum-operations-to-make-array-sum-divisible-by-k.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */

const sum=(arr)=>{
    let s=0;
    for(let val of arr){
        s+=val;
    }
    return s;
}
var minOperations = function(nums, k) {
    let totalSum=sum(nums);
    return(totalSum % k);
};